lib/runner.js

```diff
@@ -89,10 +89,11 @@ __private__.addProcess = function(number, queue) {
 
 
 __private__.shutdown = function() {
-  var endMoment = moment.utc();
+  var endMoment = moment.utc(),
+    totalDuration = moment(endMoment.diff(__private__.startDate)).format('HH:mm:ss');
```

Comment:
1. __private__.startDate is also a moment.
can you rename to have consistency?
__private__.startDateMoment

2. why do you need to wrap into moment again? moment(endMoment.diff(....))?